server/base/src/test/java/org/apache/accumulo/server/util/TableDiskUsageTest.java

```diff
@@ -75,17 +72,20 @@ public static void beforeClass() {
 
   @Test
   public void testSingleTableMultipleTablets() throws Exception {
-    final ServerContext client = EasyMock.createMock(ServerContext.class);
-    final Scanner scanner = EasyMock.createMock(Scanner.class);
-    mockScan(client, scanner, 1);
+    final ClientContext client = EasyMock.createMock(ClientContext.class);
```

Comment:
The more levels of mocking there is in test code, the harder it is to refactor code the cod under test.   Sometimes its better to change the code to make it easy to test and to minimize the need for mocking.   For example maybe could have changed :

```java
 public static Map<SortedSet<String>,Long> getDiskUsage(Set<TableId> tableIds,
      AccumuloClient client) 
```

to :

```java
 public static Map<SortedSet<String>,Long> getDiskUsage(Set<TableId> tableIds,
      Function<TableId, Iterable<Map<StoredTabletFile,DataFileValue>>> tableFilesProvider) 
```

Maybe this would make the code easier to test.  Not a change for this PR as the changes w/ easy mock were already made.   just something to consider for future changes.

Reply:
Yeah I agree the mocking that's done for this test now is definitely not ideal. I did not consider doing something like that, but you're right it would have made things a lot simpler for the test here. I'll be keeping this in mind for future changes, thanks!